src/platform/windows/mod.rs

```diff
@@ -58,13 +59,14 @@ impl MediaControls {
             controls,
             display_updater,
             timeline_properties,
+            button_handler_token: None,
         })
     }
 
     /// Attach the media control events to a handler.
     pub fn attach<F>(&mut self, event_handler: F) -> Result<(), Error>
     where
-        F: Fn(MediaControlEvent) + Send + 'static,
+        F: Fn(MediaControlEvent) + Send + Sync + 'static,
```

Comment:
This is the only problem I see. In the Linux and Mac souvlaki APIs only Send is required. This can be confusing. From a design standpoint, which do you think is better, to remove all mutexes that wrap event handlers and enforce the `Sync` trait guard to the F type, or to put back the mutex and keep the trait guard as is. Or third option, make the where clause different in Windows but clarify it.

Reply:
Good point; matching the other platforms is better.